### DISCUSSION TITLE
Category Page not rendering Columns correctly

### DISCUSSION THREAD

**Discussion by 2EyeTec** [breezefront/community#117] [1. Help] [answered]
On my category pages, the 2 column layout shows the columns below each other and not next to each other .  I tried to find out why by comparing it to the demo page, but unfortunately the level of my know-how is too low .  I really would appreciate some help and I'm happy to provide more information!

It's a new installation (2.4.6) with an imported database and I was using the luma template before.

Thanks!

**Comment by vovayatsyuk** [comment]
An URL to your page would help a lot.

**Reply by 2EyeTec** [reply]
mgnt.2eyetec.com

**Reply by vovayatsyuk** [reply]
The `body` element is missing all css classes for some reason. Try to disable all third-party modules.

**Reply by 2EyeTec** [reply]
Thanks for the quick answer!
I disabled all 3rd party modules (except the swissup modules), it' still not working ...

**Reply by vovayatsyuk** [reply]
Can you also check if your "Content > Configuration > Head > Scripts and Style Sheets" doesn't contain HTML tags? It's allowed to put head-compatible tags only (script, style, meta).

**Reply by vovayatsyuk** [reply, marked as answer]
Looks like something was here:

![image](https://user-images.githubusercontent.com/306080/235691062-80626e41-d8ca-4c43-898f-e6b8060f5cc4.png)

**Reply by 2EyeTec** [reply]
Okay - thanks for the hint! Found it, removed it and now it's working! Thanks a lot for your help!
